jabgui/src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -374,13 +373,18 @@ private void initKeyBindings() {
     }
 
     private void initBindings() {
-        // Bind global state
-        FilteredList<Tab> filteredTabs = new FilteredList<>(tabbedPane.getTabs());
-        filteredTabs.setPredicate(LibraryTab.class::isInstance);
-
         // This variable cannot be inlined, since otherwise the list created by EasyBind is being garbage collected
-        openDatabaseList = EasyBind.map(filteredTabs, tab -> ((LibraryTab) tab).getBibDatabaseContext());
-        EasyBind.bindContent(stateManager.getOpenDatabases(), openDatabaseList);
+        openDatabaseList = EasyBind.map(tabbedPane.getTabs(), tab -> {
+            if (tab instanceof LibraryTab libraryTab) {
+                return libraryTab.getBibDatabaseContext();
+            } else {
+                return null;
+            }
+        });
+
+        // call compromised until further notice, See https://github.com/JabRef/jabref-koppor/pull/713 for details.
```

Comment:
I cased wrong - new proposal

```suggestion
        // Call compromised until further notice. See https://github.com/JabRef/jabref-koppor/pull/713 for details.
```